grails-app/services/org/pih/warehouse/product/ProductMergeService.groovy

```diff
@@ -140,6 +191,7 @@ class ProductMergeService {
             // Swap assemblyProduct to primary
             obsoletedComponent.assemblyProduct = primary
             // Note: needs flush because of "User.locationRoles not processed by flush"
+            // (this was an issue on grails 1.3.9, might not be a case anymore)
```

Comment:
Maybe it will be worth creating a ticket for checking that and removing if this is unnecessary?

Reply:
Added it to this one https://pihemr.atlassian.net/browse/OBPIH-5484